tests/integration/test_notebooks_gpu.py

```diff
@@ -41,7 +42,7 @@ def test_ncf_integration(notebooks, size, epochs, expected_values):
         OUTPUT_NOTEBOOK,
         kernel_name=KERNEL_NAME,
         parameters=dict(
-            TOP_K=10, MOVIELENS_DATA_SIZE=size, EPOCHS=epochs, BATCH_SIZE=512
+            TOP_K=10, MOVIELENS_DATA_SIZE=size, EPOCHS=epochs, BATCH_SIZE=512, SEED=SEED,
```

Comment:
I would argue that seed should be a parameters, we are getting the metrics we get because we have seed=42, right? with another seed we will get different metrics. Would this make sense?

FYI @yueguoguo @anargyri @gramhagen

After reviewing this, we should rethink if we need SEED in the constants

Reply:
You mean, we should directly put 42 instead of using `constant.SEED = 42`?

Reply:
yeah

Reply:
Changed to set `42` via test parameters in integration tests.